File: 2025/학부연구생/Code/Python programming_grade/class_v2.2/2025_python_Answer_re/problem3.py
# input.txt 파일에서 입력 읽기
with open('input.txt', 'r') as f:
    n = int(f.read().strip())

# 계단 형태로 별 출력하기
result = ""
for i in range(1, n + 1):
    result += "*" * i
    if i != n:  # 마지막 줄이 아니면 줄바꿈 추가
        result += "\n"

# output.txt 파일에 결과 쓰기
with open('output.txt', 'w') as f:
    f.write(result)

File: 2025/학부연구생/Code/Python programming_grade/class_v2.2/C_answer_Code/problem3.cpp
#include <iostream>
#include <fstream>

int main() {
    // Declare a variable to store the input value N
    int N;

    // Create input and output file streams
    std::ifstream inputFile("input.txt");
    std::ofstream outputFile("output.txt");

    // Read the integer N from the input file
    inputFile >> N;

    // Loop to print the pyramid with stars
    for (int i = 1; i <= N; i++) {
        // Print spaces before stars
        for (int j = 1; j <= N - i; j++) {
            outputFile << " ";
        }
        // Print stars
        for (int k = 1; k <= 2 * i - 1; k++) {
            outputFile << "*";
        }
        outputFile << "\n";
    }

    // Close the file streams
    inputFile.close();
    outputFile.close();

    return 0;
}